apps/session-manager/src/services/redis-session-storage.ts

```diff
@@ -294,6 +294,13 @@ const getUserTokens = (
   },
 });
 
+const noneOrErrorWhenNotFound: <T>(
```

Comment:
❤️ `getByZendeskToken` can be refactored as well

Reply:
Fixed in 2bbd7e8